exercises/accumulate/runtests.jl

```diff
@@ -0,0 +1,25 @@
+# canonical data version: 1.1.0
+
+ using Test
+
+ include("accumulate.jl")
+
```

Comment:
The description uses squares as an example, it would be a good candidate for a test that uses an anon function and not a pre-defined operator.

Reply:
I think I addressed this